app/controllers/jobs_controller.rb

```diff
@@ -1,6 +1,47 @@
 class JobsController < ApplicationController
   def index
-    jobs = Job.filter_by_work_type(params[:work_type])
+    jobs = filter_jobs_by_work_type
+    jobs = filter_jobs_by_location(jobs) if params[:location].present?
+    jobs = filter_jobs_by_keyword(jobs) if params[:keyword].present?
+
     @pagy, @jobs = pagy(jobs, limit: Settings.jobs.page_size)
   end
+
+  private
+
+  def filter_jobs_by_work_type
+    if params[:work_type].present?
+      Job.filter_by_work_type(params[:work_type])
+    else
+      Job.all
+    end
```

Comment:
```suggestion
      Job.filter_by_work_type(params[:work_type])
```
Để lại mỗi như này thôi là được
Em chỉnh lại tương tự đối với các tiêu chí khác nhé